packages/mobile/src/screens/edit-track-screen/components/FormScreen.tsx

```diff
@@ -32,6 +33,26 @@ export const FormScreen = (props: FormScreenProps) => {
   const { children, bottomSection, style: styleProp, ...other } = props
   const styles = useStyles()
   const navigation = useNavigation()
+  const [hideBottomSection, setHideBottomSection] = useState(false)
+
+  // // On android, explicitly hide bottomSection when keydoard shown.
+  // // This will prevent users from accidently pressing "continue".
+  useLayoutEffect(() => {
+    if (Platform.OS === 'android') {
+      const showSubscription = Keyboard.addListener('keyboardDidShow', () => {
```

Comment:
Probably not useful in this case but we do have the keyboard state in the store

Reply:
oh yeah good call, let me check

Reply:
its much cleaner now, great call